services/QuillLMS/client/app/bundles/Evidence/components/studentView/readAndHighlightInstructions.tsx

```diff
@@ -4,14 +4,25 @@ import DisplayStudentHighlight from './displayStudentHighlight'
 import DirectionsSection from './directionsSection'
 
 import { helpIcon, Tooltip } from '../../../Shared/index'
+import useFocus from '../../../Shared/hooks/useFocus'
 
 const ReadAndHighlightInstructions = ({ passage, activeStep, studentHighlights, removeHighlight, inReflection, }) => {
+  const [containerRef, setContainerFocus] = useFocus()
+
+  React.useEffect(() => {
+    setContainerFocus()
+  }, [])
+
+  React.useEffect(() => {
+    setContainerFocus()
+  }, [inReflection])
```

Comment:
Do we want both of these declarations?  Seems like the first one would mean that our attempt to not run it all the time in the second one would fail, or am I missing something?

Reply:
Aha!  I didn't know that's what the empty array did.  Interesting, and sort of weird way of handling it, but that makes sense to me!